app/src/main/kotlin/cz/covid19cz/erouska/service/CovidService.kt

```diff
@@ -254,6 +254,7 @@ class CovidService : Service() {
     }
 
     private fun startBleScanning() {
+        bleScanningDisposable?.dispose()
```

Comment:
udelas to i pro advertising? sice to tam je min pravdepodobny..ale at je to konzistentni